Doppler.Sap/appsettings.json

```diff
@@ -23,7 +23,8 @@
           "Endpoint": "BusinessPartners"
         },
         "BillingConfig": {
-          "Endpoint": "Orders"
+          "Endpoint": "Orders",
+          "NeedCreateIncomingPayments": "false"
```

Comment:
You are using a string for a boolean value.

```suggestion
          "NeedCreateIncomingPayments": false
```

Reply:
yes, my bad. I will update it.

Reply:
Done